sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/DigitalTwinsClientSample.csproj

```diff
@@ -17,7 +17,7 @@
     <PackageReference Include="CommandLineParser" Version="2.7.82" />
     <PackageReference Include="Azure.Storage.Blobs" Version="12.16.0" />
 
-    <PackageReference Include="System.Text.Json" Version="6.0.11" />
+    <PackageReference Include="System.Text.Json" Version="8.0.5" />
```

Comment:
Can we please add a follow-up issue to review either removing this entirely or converting to use the central dependencies?

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/49123